study.md

```diff
@@ -67,5 +67,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+Client sends GET to server, server sends GET to controller, controller sends GET to model, model profores GET request then sends back to server where server processes GET and sends back to client. 
```

Comment:
The client sets a GET request and URL to the server.  The router decides which controller action to execute based on the URL and the verb.  The controller action executes, if a route for the URL and verb exists.  The controller method may or may not invoke methods on a model, depending on the method.  Assuming that it does interact with the model layer and the model performs a database action, the controller can take the model data that has been returned and render it as a response to the client.